test/test_drivers.jl

```diff
@@ -72,3 +72,10 @@ end
         )
     end
 end
+
+@testset "Test Driver from Extension" begin
```

Comment:
Might it be possible to add tests for the error path? It'll be good to check that the error message will be rendered as intended.